### DISCUSSION TITLE
Cost Management with Gruntwork Modules

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#453] [Help] [answered]
The Gruntwork [Production Framework](https://docs.gruntwork.io/guides/production-framework) is something our organization is using as a guide to building our environment with best practices. Is there anything in the Gruntwork repositories that can help me manage my AWS costs? 

---

<ins datetime="2022-06-09T14:12:14Z">
  <p><a href="https://support.gruntwork.io/hc/requests/108735">Tracked in ticket #108735</a></p>
</ins>

**Comment by zackproser** [comment, marked as answer]
Great question. While there's nothing currently within the Gruntwork IaC library that can specifically help you manage your costs, we recommend taking advantage of the [AWS Budgets service](https://aws.amazon.com/aws-cost-management/aws-budgets/), so that you can configure alert thresholds for your AWS costs and get a granular breakdown of how the various resources in your account contribute to your total bill.
